src/main/kotlin/org/rust/lang/core/RsAstFactory.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core
+
+import com.intellij.core.CoreASTFactory
+import com.intellij.psi.impl.source.tree.LeafElement
+import com.intellij.psi.impl.source.tree.PsiCommentImpl
+import com.intellij.psi.tree.IElementType
+import org.rust.lang.core.psi.RS_DOC_COMMENTS
+import org.rust.lang.core.psi.RsDocCommentImpl
+
+class RsAstFactory : CoreASTFactory() {
```

Comment:
Let's use a common naming convention for such classes and rename it to `RsASTFactory ` as all other inheritances of `ASTFactory` do